### PR TITLE
Feature DOM Events course

### PR DESCRIPTION
**Pull request by laurosilvacom**
Update new section to feature DOM events course. 

image

image

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -16,8 +16,7 @@ const IndexPage: FunctionComponent = ({homePageData}: any) => {
         openGraph={{
           images: [
             {
-              url:
-                'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1617697492/next.egghead.io/cards/egghead.io-digital-garden-cli-with-rust_2x.png',
+              url: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1621135706/egghead-next-pages/dom-events/main-ogImage-dom-events_2x.png',
```
Comment (laurosilvacom) [inline src/pages/index.tsx:19]: Feature course in the index OgImage:

image

**Comment by wjohnson85** [comment]
For the hero images could we link to the instructors page instead of linking offsite to their Twitter account?

**Comment by zacjones93** [comment]
> For the hero images could we link to the instructors page instead of linking offsite to their Twitter account?

This should be solved in a separate PR as the Jumbotron component is coded to pull in the instructor's path which is set to their website 

https://github.com/eggheadio/egghead-next/blob/596678ac4dabcc4c1727a62ad4533419bd95b290/src/pages/new/index.tsx#L106

File: src/pages/new/index.tsx [hunk 2]
```diff
@@ -77,11 +77,11 @@ const CourseFeatureCard = ({resource, className}: any) => {
               <p className="text-sm mt-4">{description}</p>
             </div>
           </div>
-          <img
+          {/* <img
             className="absolute top-0 left-0 z-0 w-full"
             src={featureCardBackground}
             alt=""
-          />
+          /> */}
```
Comment (laurosilvacom) [inline src/pages/new/index.tsx:84]: Remove the feature card background. Less noise. Focus on the hero banner. 

image

vs.

image

**Comment by vojtaholik** [comment]
couple nitpicks :)

nitpicks@2x

**Comment by laurosilvacom** [comment]
Updated padding and title (with Joel's input) 

image

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**